### ISSUE TITLE
Workshop Editor Issues

### ISSUE THREAD

**Issue by themisfitgames** [SmartlyDressedGames/Unturned-3.x-Community#511]
Hello @SDGNelson 
I've noticed as of recently within the past few updates, now when you go to select objects in the workshop editor to move them it selects all of the other items in that general vicinity besides what you specifically want chosen. I'm not sure if it's a setting error on my end or if it's a bug within unturned itself now, but it's taking me significantly longer to create/edit workshop maps, and it's causing several mistakes.

Per example: When you select an object on the screen in workshop editor instead of the selector tool being from left to right as a swipe while holding the left mouse button; the mouse cursor stops halfway in the middle of that box to make a finished selection rather than being on the lower right-hand side.

Attached are a few images attempting to explain my issue, any ideas as to what would be the cause?
![unturnedissue_1](https://user-images.githubusercontent.com/20868143/50333145-2f44a700-04b9-11e9-8ac1-da3fcdc26823.png)
![unturnedissue_2](https://user-images.githubusercontent.com/20868143/50333147-2f44a700-04b9-11e9-8672-0bbfec2192d1.png)

?

Thanks!

**Comment by SDGNelson** [comment]
Hmm my guess is that it's throwing an exception partway through the drag. After causing that to happen could you upload the output_log.txt from Unturned_Data here please? Should be straightforward to fix with the log.

**Comment by themisfitgames** [comment]
Hello @SDGNelson attached is the output_log.txt as you requested, and yikes, the amount of errors in the log.

Thank you for the quick response! 

[output_log.txt](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/files/2704762/output_log.txt)

**Comment by SDGNelson** [comment]
I'm surprised that the log doesn't contain any exceptions, but it does mention a bunch of missing objects. I'll see if the box selection code properly handles missing objects

**Comment by SDGNelson** [comment]
Sorry this issue is still open! Experimented with forcing some object errors but wasn't able to replicate the drag-select problem. I wonder if you could send over the map file and I'll see if it happens on my PC?

**Comment by SDGNelson** [comment]
Any luck after 3.28 released?

**Comment by themisfitgames** [comment]
Yes sir, a lot less laggy to be quite honest, and the map editor seems to navigate a bit smoother.
